docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,90 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
+
+## Prerequisites
+
+To use the **Slack connector** the following prerequisites need to be set-up. Read the [appendix](#appendix) to find out more.
+
+1. [Slack workspace](#use-a-well-known-slack-workspace) - this is the workspace the **Slack connector** will communicate with
+2. [Slack basic app with bot token configured](#configure-a-basic-slack-app) - the **Slack connector** will communicate through this Slack app with the workspace - you can consider the Slack app as _Slack bot representing the Camunda platform_
+3. [Slack bot token stored as secret](#store-slack-bot-token-as-secret) - the secret will store the Slack bot token and can be used to reference it from BPMN processes without revealing it in the BPMN `xml`
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, follow the steps below:
+
+1. Open Web Modeler and create a new BPMN diagram.
+2. Add a **Task** and an **EndEvent**.
+3. Click on the **Task**, click the wrench icon, and select which **Slack connector** you want to use. Currently Camunda 8 only supports to send messages to channels or individuals using the **Slack: Send Message connector**
+
+![create slack connector wrench](../img/connectors-slack-create-task-wrench.png)
+
+Alternatively, you can directly create a **Slack connector** by using the append menu:
+
+![create slack connector append](../img/connectors-create-task-append.png)
+
+## Make your Slack connector executable
+
+To make the **Slack connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![slack connector red properties](../img/connectors-slack-red-properties.png)
+
+All the mandatory and non-mandatory fields will be covered in the upcoming sections.
+
+### Authentication
+
+The **Slack connector** uses an OAuth bearer token (ie. the Slack app bot token) to authentication with the Slack API. Please follow along [the guide in the appendix](#store-slack-bot-token-as-secret) for how to store your Slack bot token as Secret. You can then reference the token by configuring for example `secrets.SLACK_OAUTH_TOKEN` in the **OAuth Token** field in the properties panel:
+
+![slack connector oauth token](../img/connectors-slack-oauth-token.png)
+
+Please note that the token needs to have at least the following Slack scopes configured in order for the **Slack connector** to work:
+
+- **Slack Send Message connector**: [`chat:write`](https://api.slack.com/scopes/chat:write)
```

Comment:
This belongs to the appendix I believe. 

Given that the slack app is properly configured the secret should just work.

Reply:
Yes